src/attributes/components/AttributeDetails/messages.tsx

```diff
@@ -0,0 +1,260 @@
+import React from "react";
+import { defineMessages, FormattedMessage } from "react-intl";
+
+export const messages = defineMessages({
+  attributeLabel: {
+    defaultMessage: "Default Label",
+    description: "attribute's label"
+  },
+  attributeSlug: {
+    defaultMessage: "Attribute Code",
+    description: "attribute's slug short code label"
+  },
+  attributeSlugHelperText: {
+    defaultMessage: "This is used internally. Make sure you don’t use spaces",
+    description: "attribute slug input field helper text"
+  },
+  entityType: {
+    defaultMessage: "Entity",
+    description: "attribute's editor component entity"
+  },
+  inputType: {
+    defaultMessage: "Catalog Input type for Store Owner",
+    description: "attribute's editor component"
+  },
+  valueRequired: {
+    defaultMessage: "Value Required",
+    description: "check to require attribute to have value"
+  }
+});
+
+export const inputTypeMessages = defineMessages({
+  dropdown: {
+    defaultMessage: "Dropdown",
+    description: "product attribute type"
+  },
+  file: {
+    defaultMessage: "File",
+    description: "file attribute type"
+  },
+  multiselect: {
+    defaultMessage: "Multiple Select",
+    description: "product attribute type"
+  },
+  references: {
+    defaultMessage: "References",
+    description: "references attribute type"
+  },
+  text: {
+    defaultMessage: "Text",
+    description: "text attribute type"
+  },
+  numeric: {
+    defaultMessage: "Numeric",
+    description: "numeric attribute type"
+  }
+});
+
+export const selectUnit = (
+  <FormattedMessage
```

Comment:
I would rather group these messages somehow together into `defineMessages` instead of making each as `FormattedMessage` component here, what do you think?

Reply:
```tsx
export const unitMessages = defineMessages({...});
export const unitSystemMessages = defineMessages({...});
export const unitTypeMessages = defineMessages({...});
```

Reply:
Personally I hate using `intl.formatMessage` and being force to import that hook so I use it as rare as I can. 
But I've grouped units to a simple object as there are a lot of them.

Reply:
I agree with @orzechdev. I think exporting Formatted messages as components is creating unnecessary boiler plate. The fact that now this file needs comments just to be able to navigate between those messages, whereas with defineMessages it's very straightforward I'd say adds to that feeling. We recently discussed strategy for messages in saleor frontend team and we set up on what @orzechdev mentioned so please stick to that.